test/system/130-kill.bats

```diff
@@ -70,6 +81,10 @@ load helpers
     run_podman wait $cid
     run_podman rm $cid
     wait $podman_log_pid
+
+    if [[ $_image != $IMAGE ]]; then
+        run_podman rmi $_image
+    fi
```

Comment:
I'm not quite sure why you need to conditionalize this, couldn't you just always do the `rmi`?

Reply:
No, that would be un-nice. The entire rationale behind this test suite is to minimize pulling. In order to do so, all tests are guaranteed to have one and exactly one `$IMAGE` loaded at start-time; so it is courteous for each test to (absent any failures) leave state as they found it.